To: vim-dev@vim.org
Subject: Patch 7.3.018
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.018 (after 7.3.012)
Problem:    Missing argument to windres in MingW makefiles.
Solution:   Add the argument that was wrapped in the patch. (Jon Maken)
Files:	    src/Make_ming.mak, src/GvimExt/Make_ming.mak


*** ../vim-7.3.017/src/Make_ming.mak	2010-09-29 12:37:53.000000000 +0200
--- src/Make_ming.mak	2010-09-29 18:37:08.000000000 +0200
***************
*** 303,309 ****
  endif
  endif
  CC := $(CROSS_COMPILE)gcc
! WINDRES := $(CROSS_COMPILE)windres --preprocessor="$(CC) -E -xc" -
  
  #>>>>> end of choices
  ###########################################################################
--- 303,309 ----
  endif
  endif
  CC := $(CROSS_COMPILE)gcc
! WINDRES := $(CROSS_COMPILE)windres --preprocessor="$(CC) -E -xc" -DRC_INVOKED
  
  #>>>>> end of choices
  ###########################################################################
*** ../vim-7.3.017/src/GvimExt/Make_ming.mak	2010-09-29 12:37:53.000000000 +0200
--- src/GvimExt/Make_ming.mak	2010-09-29 18:36:39.000000000 +0200
***************
*** 33,39 ****
  endif
  endif
  CXX := $(CROSS_COMPILE)g++
! WINDRES := $(CROSS_COMPILE)windres --preprocessor="$(CXX) -E -xc" -
  LIBS :=  -luuid
  RES  := gvimext.res
  DEFFILE = gvimext_ming.def
--- 33,39 ----
  endif
  endif
  CXX := $(CROSS_COMPILE)g++
! WINDRES := $(CROSS_COMPILE)windres --preprocessor="$(CXX) -E -xc" -DRC_INVOKED
  LIBS :=  -luuid
  RES  := gvimext.res
  DEFFILE = gvimext_ming.def
*** ../vim-7.3.017/src/version.c	2010-09-29 18:32:47.000000000 +0200
--- src/version.c	2010-09-29 18:41:53.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     18,
  /**/

-- 
hundred-and-one symptoms of being an internet addict:
231. You sprinkle Carpet Fresh on the rugs and put your vacuum cleaner
     in the front doorway permanently so it always looks like you are
     actually attempting to do something about that mess that has amassed
     since you discovered the Internet.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
